platform/viewer/src/components/Header/Header.css

```diff
@@ -97,26 +99,45 @@
   margin-right: 1rem;
 }
 
-.header-versionInfo {
```

Comment:
Why is this being removed? Are we gonna keep the version info only in the about modal?

Reply:
This is a request from @swederik.
Yes, we'll still have the version in the About Modal, but it will no longer be visible in the header.